sync/models/sync_trigger_cron.py

```diff
@@ -20,11 +21,9 @@ class SyncTriggerCron(models.Model):
         "ir.cron", delegate=True, required=True, ondelete="cascade"
     )
 
-    @api.model_create_multi
-    def create(self, vals_list):
-        for vals in vals_list:
-            vals.setdefault("name", vals.get("trigger_name", "Sync"))
-        return super(SyncTriggerCron, self).create(vals_list)
+    def create(self, vals):
```

Comment:
Always use ``@api.model`` for create method

https://odoo-source.com/?q=def%20create%5C(self%2C%20vals%5C)&i=nope&files=&excludeFiles=po%24%7Cpot%24%7Cyml%24%7Cyaml%24%7Ccss%24%7C%2Fstatic%2Flib%2F&repos=13.0